File: 9/solution.go
package main

import "fmt"
import "math"

func main() {
	for i := 3;i<500;i++ {
		for j:=i+1;j<500;j++ {
			k := math.Sqrt(float64(i*i+j*j))
			if k == float64(int(k)) && i+j+int(k)==1000 {
				fmt.Printf("%d\n", i*j*int(k))
			}
		}
	}
}

File: 9/solution.py
from random import random
from math import *

for i in range(3, 500):
	for j in range(i+1, 500):
		k = sqrt(i*i+j*j)
		if k == int(k) and i+j+k==1000:
			print(i*j*int(k))
